src/DistributedMetadata/DDLService.cpp

```diff
@@ -589,4 +650,23 @@ void DDLService::processRecords(const IDistributedWriteAheadLog::RecordPtrs & re
     /// FIXME, update DDL task status after committing offset / local offset checkpoint ...
 }
 
+std::vector<Poco::URI> DDLService::getTargetURIs(
+    const std::unordered_map<String, String> & headers, const String & database, const String & table, const String & method) const
+{
+    const auto & column_ddl_request = headers.contains("column");
```

Comment:
`auto column_ddl_request = headers.contains("column")`. btw, we already have the OpCode, why we need another way to decide if it ia column ddl request ? We shall use record OpCode to decide if it is a column or table ddl. We can get rid of `column` from the record header after introducing the OpCode

Reply:
No, headers.at("column") will be used in getColumnApiPath. So here we can decide column  by headers.contains("column"), Without having to judge  if( OpCode  == CREATE_COLUMN | ALTER_COLUMN | DELETE_COLUMN)

Reply:
I would suggest using the latter like the following code does. The reason is opcode is defined for this purpose

```
bool column_ddl = record->opcode == CREATE_COLUMN || ...
```
```